.gitignore

```diff
@@ -44,10 +44,11 @@ ltdl.m4
 ltmain.sh
 missing
 py-compile
-m4/ltoptions.m4
-m4/ltsugar.m4
-m4/ltversion.m4
-m4/lt~obsolete.m4
+/m4/ltargz.m4
+/m4/ltoptions.m4
+/m4/ltsugar.m4
+/m4/ltversion.m4
+/m4/lt~obsolete.m4
```

Comment:
But then we will lost track of new additions, which should perhaps
be reflected in `make distclean` etc. as well.  Hmm, I will check if
it should not be added here, too.